portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,16 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+
+/**
+ * The above code is organized to show each individual step, but we can use an
+ * ES6 feature called arrow functions to shorten the code. This function
+ * combines all of the above code into a single Promise chain. You can use
+ * whichever syntax makes the most sense to you.
+ */
+function getHellofromFetchedPractice() {
+  fetch('/data').then(response => response.text()).then((quote) => {
+    document.getElementById('fetch-container').innerText = quote;
+  });
+}
```

Comment:
Does this work?
Also the comments here are a little confusing, can you rephrase them?

Reply:
Yes I think this method works fine on my portfolio. I have added some further comments there on the newest version